cmd/root/auth.go

```diff
@@ -195,6 +195,12 @@ func MustWorkspaceClient(cmd *cobra.Command, args []string) error {
 		cfg.Profile = profile
 	}
 
+	_, isTargetFlagSet := targetFlagValue(cmd)
+	if !isTargetFlagSet && hasProfileFlag {
+		// If the profile flag is set but the target flag is not, we should skip loading the bundle configuration.
+		cmd.SetContext(SkipLoadBundle(cmd.Context()))
```

Comment:
What if there's an API someday with a field/name `target`? We would run into trouble because normally, `--target` is scoped to bundle commands, but because of this, we end up resolving bundle auth.

I think we should always not load bundle configuration if:
1. The `--profile` flag is specified.
2. It's a non-bundle command.

WDYT?

Reply:
Thanks, makes sense.